examples/recursive-src/.merlin

```diff
@@ -0,0 +1,30 @@
+# This file is autogenerated for
+# [Merlin](https://github.com/the-lambda-church/merlin), a static analyser for
+# OCaml that provides autocompletion, jump-to-location, recoverable syntax
+# errors, type errors detection, etc., that your editor can use. To activate it,
+# one usually provides a .merlin file at the Path.the_root of a project, describing where
```

Comment:
I think you accidentally caught the `root` here when you did a previous refactor.